README.md

```diff
@@ -122,6 +119,23 @@ ChefSpec::SoloRunner.new(log_level: :debug).converge(described_recipe)
 
 **NOTE** You do not _need_ to specify a platform and version to use ChefSpec. However, some cookbooks may rely on [Ohai](http://github.com/chef/ohai) data that ChefSpec cannot not automatically generate. Specifying the `platform` and `version` keys instructs ChefSpec to load stubbed Ohai attributes from another platform using [fauxhai](https://github.com/customink/fauxhai).
 
+### ChefZero Server
+
+The `ServerRunner` uses a [chef-zero](https://github.com/chef/chef-zero) instance as a stand-in for a full Chef Server. The instance is created at the initiation of the ChefSpec suite and is terminated at its completion. In between each test the state of the ChefZero server is completely reset.
+
+```ruby
+RSpec.configure do |config|
+  # When using ChefSpec::ServerRunner, specify the data storage method (options: in_memory, on_disk; default: in_memory)
+  # If you are in a low-memory environment, setting this value to :on_disk may improve speed and/or reliability.
+  config.server_runner_data_store = :on_disk
```

Comment:
So, in the comment you say that the default is `in_memory` but you set here `on_disk`. 🤔